Make the threshold for predictive echoing configurable (#1518)
wez/wezterm: docs/config/lua/SshDomain.md

}
```

> You main now specify the round-trip latency threshold for enabling predictive

Reviewer: ```suggestion
You maynow specify the round-trip latency threshold for enabling predictive
```

Reviewer 2: Or even more correct 😅
```suggestion
You may now specify the round-trip latency threshold for enabling predictive
```

> You main now specify the round-trip latency threshold for enabling predictive

Reviewer: I think a similar thing should be added to the TlsDomain docs as well!

local echo using `local_echo_threshold_ms`. If the measured round-trip latency
between the wezterm client and the server exceeds the specified threshold, the
client will attempt to predict the server's response to key events and echo the
result of that prediction locally without waiting, hence hiding latency to the

> user.

Reviewer: ```suggestion
user.  This option only applies when `multiplexing = "WezTerm"`.
```
